otter/test/test_retry.py

```diff
@@ -456,13 +458,19 @@ class EffectfulRetryTests(SynchronousTestCase):
     # an object which calls a function with arguments specified as public
     # attributes, I guess...
 
+    def setUp(self):
+        self.dispatcher = ComposedDispatcher([
```

Comment:
I'm not sure what you mean about a global dispatcher. are you talking about the functions in otter.effect_dispatcher? Did we have a global dispatcher somewhere?

My intention is for the ideal to be having one main dispatcher in an actual running program, but not for it to be a module-global variable -- more like a local variable in something like main() or a REST handler.